handler/pingpong.go

```diff
@@ -0,0 +1,21 @@
+package handler
+import (
+	"../interfaces"
+	"strings"
+	"errors"
+)
+
+// Very simple ping-pong handler.
+type PingPongHandler struct {}
+
+// Handling a message.
```

Comment:
comment on exported method PingPongHandler.DoHandle should be of the form "DoHandle ..."